src/behaviors/random-channel.js

```diff
@@ -0,0 +1,36 @@
+import Behavior from './behavior.js';
+
+class RandomChannel extends Behavior {
+  constructor(settings = {}) {
+    settings.name = settings.name || 'Random Channel';
```

Comment:
This can just be `settings.name = 'Random Channel';` unless you anticipate someone extending RandomChannel for some reason, but it's not really set up for it.